include/eve/module/core/regular/impl/simd/riscv/simd_cast.hpp

```diff
@@ -0,0 +1,183 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
+{
+
+template<plain_scalar_value T, typename N>
+EVE_FORCEINLINE wide<T, typename N::combined_type>
+                riscv_lmul_ext(wide<T, N> a) noexcept
```

Comment:
Are you running clang format? I'm pretty sure it'd get reformatted.

It's ok - if you don't, then there will just be a reformatting patch later.

Reply:
Yep, I will run git-clang-format later

Reply:
Formatted